apps/barista/src/pages/single-page/single-page.ts

```diff
@@ -32,14 +32,15 @@ export class BaSinglePage implements BaPage, AfterViewInit {
   }
   set contents(value: BaSinglePageContent) {
     this._contents = value;
-    this._recentlyOrderedService.saveToLocalStorage(this.contents);
+    if (this.contents && this.contents.layout !== 'icon') {
```

Comment:
Please check for `BaLayoutType.Icon` here instead of `'icon'`. (It's part of the shared barista-definitions).